### PR TITLE
ED-1521 checkbox visual

### PR DESCRIPTION
**Pull request by stevenburnell-dit**

### CODE REVIEW

**Comment by richtier** [comment]
linting error: 
image

File: enrolment/static/sass/main.scss [hunk 1]
```diff
@@ -326,12 +326,28 @@ label, input, button, select, textarea {
     line-height: 1.25;
     margin-bottom: 10px;
   }
-
+  /*
```
Comment (richtier) [inline enrolment/static/sass/main.scss]: please remove code we dont want to keep

File: enrolment/tests/test_forms.py [hunk 2]
```diff
@@ -61,32 +58,18 @@ def test_company_form_fields():
...
-    assert form.is_valid()
-
-
 def test_company_export_status_form_validars():
     field = forms.CompanyExportStatusForm.base_fields['export_status']
     validator = shared_validators.export_status_intention
     assert validator in field.validators
 
 
+def test_company_export_status_terms_agreed_checlbox_widget():
```
Comment (konradko) [inline enrolment/tests/test_forms.py]: typo

Comment (stevenburnell-dit) [inline enrolment/tests/test_forms.py]: fixed

File: enrolment/static/sass/main.scss (hunk 1 above)
Comment (stevenburnell-dit) [inline enrolment/static/sass/main.scss]: removed

**Pull request merged**